userprofile/index.html

```diff
@@ -1,93 +1,129 @@
 <!DOCTYPE html>
-<html ng-app='TweetSearch'>
+<html ng-app="TweetSearch">
 
 <head>
+    <meta charset="utf-8">
     <link rel="icon" type="image/png" href="../../artwork/favicon.png">
     <title>User Profile Page</title>
     <link href="/css/bootstrap.min.css" rel="stylesheet">
     <link href="/css/loklak.css" rel="stylesheet">
-	<script src="/js/angular.min.js"></script>
-	<script src="/js/jquery.min.js"></script>
-	<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/bootcards/1.0.0/css/bootcards-desktop.min.css">
-    
+<<<<<<< HEAD
+    <script src="/js/angular.min.js"></script>
+    <script src="/js/jquery.min.js"></script>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/bootcards/1.0.0/css/bootcards-desktop.min.css">
+
+    <script src="../../js/angular.min.js"></script>
+    <script src="../../js/angular-route.min.js"></script>
+    <script src="../../js/loklak.js"></script>
+    <style>
+    table, th, td {
+       border: 1px solid black;
+    }
+    </style>
+</head>
+
+<body>
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+=======
+        <script src="/js/angular.min.js"></script>
+        <script src="/js/jquery.min.js"></script>
+        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/bootcards/1.0.0/css/bootcards-desktop.min.css">
     <script src="../../js/angular.min.js"></script>
     <script src="../../js/angular-route.min.js"></script>
     <script src="../../js/loklak.js"></script>
-	<style>
-	table, th, td {
-	   border: 1px solid black;
-	}
-	</style>
+        <style>
+           table, th, td {
+           border: 1px solid black;
+         }
+        </style>
 </head>
 
 <body>
-	<nav class="navbar navbar-inverse navbar-fixed-top">
+        <nav class="navbar navbar-inverse navbar-fixed-top">
+>>>>>>> 0f6699c... Fixed html lint errors
       <div class="container-fluid">
         <div class="navbar-header">
-          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+          <button type="button" class="navbar-toggle collapsed"
+                  data-toggle="collapse" data-target="#navbar"
+                  aria-expanded="false" aria-controls="navbar">
             <span class="sr-only">Toggle navigation</span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="navbar-brand" href="http://loklak.org"><img src="/images/loklak_org.png" height="24" style="float:left;">: User Profile</a>
+<<<<<<< HEAD
+          <a class="navbar-brand" href="http://loklak.org">
+              <img src="/images/loklak_org.png" height="24" style="float:left;">: User Profile
+          </a>
+=======
+          <a class="navbar-brand" href="//loklak.org"><img src="/images/loklak_org.png" height="24" style="float:left;">: User Profile</a>
```

Comment:
Line is longer than allowed. (138 > 129)

*LineLengthBear, severity NORMAL, section `default`.*